extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Feature class",
+  "type": "object",
+  "description": "A feature class defining the properties that distinguish one set of features from another.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the feature class, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "semantic": {
```

Comment:
@lilleyse and I talked about this, we think `semantic` should be removed now that most things are dictionary-based rather than array-based.

Using dictionaries is easier to read than using indices, and it has an added benefit of enforcing a uniqueness constraint on the IDs.